workflow/apm/services.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: APM Dashboard - Services
+sidebarTitle: Services
+---
+
+The APM Services Dashboard displays all of your APM services in a unified dashboard. A service groups together interconnected endpoints, queries, or jobs within your application. This is useful for contextualizing broader issues within your software infrastructure. 
+
+The dashboard landing page displays common metrics related to your APM services like requests, latency, and errors. Use the Resources, Traces, and Logs tabs to drill into your service-specific data. This dashboard is available across other APM dashboards and the log page to easily understand how data is related to other aspects of your application. 
+
+# Resources
+
+Resources are all of the elements utilized by your services such as CPU, memory, and network bandwidth. 
+
+The Resources Table displays all of your APM services in a unified table format. This table is useful for drilling down into the traces related to the resource you are investigating to find the root cause of issues related to the service you are drilling into.
+
+The Resources Table has the following filters:
+
+| __Filter__                          | __Description__ |
+| :---------                          | :-------------- |
+| __All Operations__                  | All collected services related to your APM |
+| __Most Time Consuming__             | Sorts max latency descending order |
+| __Slowest Average Response Time__   | Average response time presented in ascending order |
+| __Highest Error Rate__              | Sorts error rate in descending order |
+| __Throughput__                      | Number of requests or operations processed in calls per minute |
+| __Apdex by Most Dissatisfying__     | Application Performance Index measures the responsiveness of a service in ascending order |
+
+# Traces
+
+Traces traverse through multiple services within a distributed system to display interconnected spans that represent an operation or function performed by a service.
```

Comment:
Would recommend rewording this to be more concise. Maybe two sentences, one of that is "what is a trace" = The parent span of an entire request,  "what is it valuable for" = this list provides a high level view of where most users entry points to your service begin, and "what else can I do" = drill into each trace to see what other spans are requested in it's lifecycle.